includes/definitions/stoneos.yaml

```diff
@@ -0,0 +1,19 @@
+os: stoneos
+text: Hillstone StoneOs
+type: network
+icon: hillstone
+group: hillstone
+over:
```

Comment:
Can you format the over graphs like the other yaml files please.

Reply:
Fixed, was my yaml format plugin that changed it.